app/User.php

```diff
@@ -12,4 +12,17 @@ class User extends Authenticatable
     protected $fillable = ['name', 'email', 'github_id', 'avatar', 'token'];
 
     protected $hidden = ['password', 'remember_token'];
+
+    public function getVerifyHash()
+    {
+        return md5($this->github_id . $this->token);
```

Comment:
It may not matter, but is there any reason not to use a hash that is more destructive?

Reply:
No reason one way or another, but md5 makes me feel a little icky.